pom.xml

```diff
@@ -117,7 +118,6 @@ under the License.
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-jxr-plugin</artifactId>
-            <version>${project.version}</version>
```

Comment:
@Bukama there is a reason for failed site build ... we should use latest release not snapshot version which have not installed yet

old parent use snapshot repositories by default so it was working